.babelrc

```diff
@@ -1,14 +1,15 @@
 {
-  "presets": [
-    "@babel/preset-env",
-    "@babel/preset-react"
-  ],
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
```

Comment:
prefer the original for readability

Reply:
![image](https://user-images.githubusercontent.com/46549/92285841-36dcf700-eeba-11ea-85a5-abb9c30a3755.png)

Reply:
Not a big deal. I just prefer things on new lines to minimize changes :)